lib/team.rb

```diff
@@ -0,0 +1,31 @@
+class Team
```

Comment:
There are quite a few "magic numbers" in this class: integers that have no semantic context to tell me what they represent.  Constants are one way of removing magic numbers: https://eliotsykes.com/magic-numbers